cookbooks/bcpc-hadoop/recipes/hdfsdu_pre_deploy.rb

```diff
@@ -0,0 +1,16 @@
+# Setup HDFSDU config
+
+include_recipe 'bcpc-hadoop::hdfsdu_kerberos'
+
+hdfsdu_keytab = ::File.join(node['bcpc']['hadoop']['kerberos']['keytab']['dir'],
+  node[:bcpc]['hadoop']['kerberos']['data']['hdfsdu']['keytab'])
+hdfsdu_hdfs_user = node['hdfsdu']['hdfs_user']
+
+node.override[:hdfsdu][:service_download_url] = get_binary_server_url
+node.override[:bcpc][:hadoop][:hdfs][:dfs][:cluster][:administrators] = \
+  node[:bcpc][:hadoop][:hdfs][:dfs][:cluster][:administrators] + ',' + \
```

Comment:
Perhaps for future work node.override[:bcpc][:hadoop][:hdfs][:dfs][:cluster][:administrators][:fs] fir multilevel admin

Reply:
I can not set `node[...]=` without a level? (If I do `node.override[...]=` I may override a default, so I would rather be opinionated that the HDFS-DU user must be there regardless of what the node attribute was before? Or do you have another API in mind?